### ISSUE TITLE
[Visual requirements  -Howdy-for-pms]: Luminosity Ratio is Less than 4.5:1 for Howdy for pms static text in the page.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#15]
**User Experience**
Low vision user will face difficulty in understand and use the functionality if contrast ratio on any informative text is less than the minimum contrast ratio.

**Test Environment**
OS: Win 10 Version 1809 (OS Build 17763.379)
URL: https://hcl-szz1925.howdy.ai/dashboard
Browser: Edge
Tool Used: Colour Contrast Analyzer (CCA)
 
**Repro Steps**
Step 1: Open the URL: https://hcl-szz1925.howdy.ai/dashboard
Step 2: Navigate to "Dashboard" tab and select it.
Step 3: Navigate to "Howdy for Your PM" link and select it.
Step 4: "Howdy for Your PM" page gets opened.
Step 5: Verify the colour contrast for static text in the page. 

**Actual Result**
Luminosity Ratio is Less than 4.5:1 for Howdy for pms static text in the page and Powered by groove link also.

**Expected Result**
Luminosity Ratio should be Greater than or Equal to 4.5:1 for Howdy for pms static text in the page and Powered by Groove link .

**Mas Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
![MAS1 4 3_Howdy for pms](https://user-images.githubusercontent.com/48272005/54473621-318c7800-4800-11e9-805a-295d4131b037.png)
![MAS1 4 3_hOWDY](https://user-images.githubusercontent.com/48272005/54473623-381aef80-4800-11e9-9870-ee361fd43a35.png)

**Comment by benbrown** [comment]
This page has been deprecated and removed from the site.

**Comment by v-gasikk** [comment]
Now this page is not available. Closing this issue based on developer comment.
